clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/GoogleOperationPoller.groovy

```diff
@@ -19,13 +19,20 @@ package com.netflix.spinnaker.clouddriver.google.deploy
 import com.google.api.services.compute.Compute
 import com.google.api.services.compute.model.Operation
 import com.google.common.annotations.VisibleForTesting
+import com.netflix.spectator.api.Clock;
+import com.netflix.spectator.api.Id;
+import com.netflix.spectator.api.Registry;
 import com.netflix.spinnaker.clouddriver.data.task.Task
 import com.netflix.spinnaker.clouddriver.google.config.GoogleConfigurationProperties
 import com.netflix.spinnaker.clouddriver.google.deploy.exception.GoogleOperationException
 import com.netflix.spinnaker.clouddriver.google.deploy.exception.GoogleOperationTimedOutException
 import org.springframework.beans.factory.annotation.Autowired
+import java.util.concurrent.TimeUnit;
+
 
 class GoogleOperationPoller {
+  static final String STARTED_METRIC_NAME = "google.operationPoller.started";
+  static final String METRIC_NAME = "google.operationPoller";
```

Comment:
Could swap the order here and use `STARTED_METRIC_NAME = "${METRIC_NAME}.started"`.